### ISSUE TITLE
Unable to authorize with github

### ISSUE THREAD

**Issue by LazyBoot** [gitextensions/gitextensions#5652]
**Do you want to request a *feature* or report a *bug*?** bug

**What is the current behavior?** Unable to click the authorize button when trying to log into github. (button seems disabled)
![image](https://user-images.githubusercontent.com/3827482/47724279-a5b1ef00-dc56-11e8-9673-72726abdad9d.png)
**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
I attempted to log into github on a new install of Git Extensions, and got to the window in the screenshot above.

**What is the expected behavior?**
To be able to click the button and log in.

**Environment you encounter the issue:**

- GitExtensions version: 2.51.05
- GIT version: 
- OS version: Win 10 pro 1803
- .NET version: 4.7.2

**Did this work in previous version of GitExtensions (which)?**

**Comment by gerhardol** [comment]
Duplicate of #4800 

GitHub token still works though
(same in 3.0)

**Comment by weiserj** [comment]
I think that I have the same problem. But I do not think, that this is a duplicate of
[#4800](https://github.com/gitextensions/gitextensions/issues/4800). I can ignore the IE-Browser hint, but the "Authorize mabako" button remains disabled!!!

I have revoked the gitextensions authorization in GitHub and I have no oauth token  (I switched from Windows7 to Windows10). I use the latest versions of git(2.19.1.windows.1), gitextensions(2.51.05) and Windows10 (Version 1803).

I have no idea how to get an authorization token and I would really appreciate very much a hint to a solution!
